app/src/main/java/com/example/composetemplate/managers/MainNetworkManager.kt

```diff
@@ -7,8 +7,11 @@ import com.example.composetemplate.data.remote.base.BaseNetworking
 /**
  * This class is responsible for managing all network operations.
  * Here, we will execute network requests and perform related tasks.
+ *
+ * This class is our MainNetworkManager for all api calls.
+ * todo: check if our main network will be firebase so we can delete FirebaseNetworkManager
```

Comment:
You can remove the TODO. Because this is a template project, and even though FirebaseNetworkManager is the main one, we still want MainNetworkManager to remain as the base class for all other network services.
Also see the my comment about FirebaseNetworkManager